pycytominer/cyto_utils/load.py

```diff
@@ -144,7 +147,7 @@ def load_npz_features(npz_file, fallback_feature_prefix="DP"):
     return df
 
 
-def load_npz_locations(npz_file):
+def load_npz_locations(npz_file, location_columns=(0, 1)):
```

Comment:
oohh, I see. Can you rename this `location_columns_index`?

Alternatively, (and this might actually be better), can you provide a different argument for `location_x_col_index` and `location_y_col_index`?

Reply:
somehow a tuple argument that has this functionality smells a bit off to me.

Reply:
Resolved in https://github.com/cytomining/pycytominer/pull/218/commits/6d5eb23bcf072c7ac344ada78bbc305d6f9ece49 (we went with 2 separate arguments).